tests/phpunit/includes/storage/StoreTest.php

```diff
@@ -184,7 +184,7 @@ public function testGetPropertiesSpecial() {
 
 		$this->assertInstanceOf( '\SMW\SQLStore\SimpleListLookup', $result );
 		foreach( $result->fetchResultList() as $row ) {
-			$this->assertEquals( 2, sizeof( $row ) );
```

Comment:
Damn, just realized that it's still silly. Fixed in https://github.com/SemanticMediaWiki/SemanticMediaWiki/pull/944